Add improvements. (#2604)
wso2/docs-is: en/docs/references/evolution-of-identity-federation-standards.md

> Single sign-on (SSO) systems have become very popular since it is a very

Reviewer: ```suggestion
Single sign-on (SSO) has become very popular since it is a very
```

secure and convenient authentication mechanism. Google Apps can be taken
as the best example for a single sign-on system where users can
automatically log in to multiple web applications once they are

> authenticated at a one place using a single credential.

Reviewer: ```suggestion
authenticated once using a single set of credentials.
```

    with any of the registered ACS URLs of the service provider, then the
    Identity Server sends the response to the matched one.
    
    ```
    https://localhost:9443/samlsso?spEntityID=foo.com&acs=http://localhost:8080/foo.com/my-home.jsp
    ```
 
Optionally, you can send a `RelayState` parameter as follows:

```
https://localhost:9443/samlsso?spEntityID=foo.com&RelayState=http://localhost:8080/foo.com/my-home.jsp
```

This request will authenticate and redirect the user to the URL in the
RelayState parameter itself.

!!! info 

> 	Either you could have service provider initiated SSO only, or service provider initiated SSO and identity provider

Reviewer: ```suggestion
	You could could have service provider initiated SSO alone, or service provider initiated SSO and identity provider initiated SSO together.
```

> 	initiated SSO. You can't have identity provider initiated SSO only. By design, service provider

Reviewer: ```suggestion
	You cannot have identity provider initiated SSO alone. By design, service provider
```
